index.html

```diff
@@ -1,700 +1,654 @@
+
     <h3>avatar group</h3>
-    <div class='inline-block'>
-        <img src='https://picsum.photos/id/1074/500/500' class='avatar avatar-size-sm avatar-border-blue border-thin border-radius-circle' />
-        <img src='https://picsum.photos/id/1074/500/500' class='avatar avatar-size-sm avatar-overlap avatar-border-blue border-thin border-radius-circle' />
-        <img src='https://picsum.photos/id/1074/500/500' class='avatar avatar-size-sm avatar-overlap avatar-border-blue border-thin border-radius-circle' />
-        <img src='https://picsum.photos/id/1074/500/500' class='avatar avatar-size-sm avatar-overlap avatar-border-blue border-thin border-radius-circle' />    
-    </div>
-    
-    <h3>avatars with badge</h3>
-    <div class='inline-block pos-relative'>
-        <img src='https://picsum.photos/id/1074/500/500' class='avatar avatar-size-sm border-radius-circle' />
-        <div class='badge-green border-radius-circle badge-size-sm pos-absolute top-left-70'></div>
+    <div class='dis-inline'>
+        <img src='./assets/avatar-2.jpg' class='brd-full b-solid b-secondary img-fit-cover' />
+        <img src='./assets/avatar-2.jpg' class='brd-full b-solid b-secondary img-fit-cover avt-overlap-xlg' />
+        <img src='./assets/avatar-2.jpg' class='brd-full b-solid b-secondary img-fit-cover avt-overlap-xlg' />
+        <img src='./assets/avatar-2.jpg' class='brd-full b-solid b-secondary img-fit-cover avt-overlap-xlg' />
     </div>
 
-    <div class='inline-block pos-relative'>
-        <img src='https://picsum.photos/id/1074/500/500' class='avatar avatar-size-sm border-radius-circle' />
-        <div class='badge-yellow border-radius-circle badge-size-sm pos-absolute top-left-70'></div>
-    </div>
-    
-    <div class='inline-block pos-relative'>
-        <img src='https://picsum.photos/id/1074/500/500' class='avatar avatar-size-sm border-radius-circle' />
-        <div class='badge-red border-radius-circle badge-size-sm pos-absolute top-left-70'></div>
-    </div><br/>
-    
-    <div class='inline-block pos-relative'>
-        <img src='https://picsum.photos/id/1074/500/500' class='avatar avatar-size-sm border-radius-circle' />
-        <div class='badge-blue border-radius-circle badge-size-md pos-absolute top-left-70'></div>
+    <h3>avatar with badge</h3>
+    <div class='dis-inblock pos-relative'>
+        <img src='./assets/avatar-2.jpg' class='brd-full img-fit-cover' />
+        <div class='bg-success brd-full badge-size-s pos-absolute br-5'></div>
     </div>
 
+
+    <!-- Images component variants -->
+
+
     <h1>Images</h1>
     <p>densty switching at 900px</p>
-    <img srcset='https://picsum.photos/id/200/700/700 900w,
-    https://picsum.photos/id/200/300/300 300w' sizes="(min-width: 900px) 900px,
-    300px" alt='Hairy cow' class='img-cover' />
+    <img srcset='./assets/img.jpg 500w,
+    ./assets/img2.jpg 200w' sizes="(min-width: 900px) 500px,
+    200px" alt='Hairy cow' class='img-fit-cover' />
```

Comment:
Appreciation - Nice work @foolhardy21 , good to see that you have used density switching here.